### PR TITLE
feat: capture username in django

### PR DESCRIPTION
**Pull request by subzero10**
Related to #155.

### CODE REVIEW

File: honeybadger/contrib/django.py [hunk 1]
```diff
@@ -128,3 +129,11 @@ def process_exception(self, request, exception):
         honeybadger.notify(exception)
         clear_request()
         return None
+
+    def __setUserFromContext(self, request):
```
Comment (Kelvin4664) [inline honeybadger/contrib/django.py:133]: Can we stick to snake casing? So things stay consistent

File: honeybadger/contrib/django.py [hunk 2]
```diff
@@ -128,3 +129,11 @@ def process_exception(self, request, exception):
...
+        # in Django 1 request.user.is_authenticated is a function, in Django 2+ it's a boolean
+        if hasattr(request, 'user') and (
```
Comment (Kelvin4664) [inline honeybadger/contrib/django.py:135]: Not a blocker, but i think we'll be fine without support for django 1.

Comment (subzero10) [reply honeybadger/contrib/django.py:135]: In our CI, Django 1 is still supported so I thought it's best to support it :)

**Comment by subzero10** [comment]
@Kelvin4664 Thanks for the review! Can I get an approval please?

**Review by Kelvin4664 — APPROVED** [review]

**Comment by subzero10** [comment]
Released with `0.17.0`!

**Pull request merged**